src/Symfony/Component/Form/Tests/ChoiceList/Factory/DefaultChoiceListFactoryTest.php

```diff
@@ -854,25 +854,6 @@ public function testCreateViewFlatlabelTranslationParametersClosureReceivesValue
         $this->assertFlatViewWithlabelTranslationParameters($view);
     }
 
-    private function assertScalarListWithChoiceValues(ChoiceListInterface $list)
```

Comment:
Was not removed when legacy tests were removed 

https://github.com/symfony/symfony/commit/31eb49863ee6d192a98414fe0e5eb3242db4641f#diff-18cef2a172697b433048f25490e0ccf5bec0b85c48b16040fb9fa3cd6bcfb4c9